perception/autoware_tensorrt_bevdet/README.md

```diff
@@ -0,0 +1,69 @@
+# tensorrt_bevdet <!-- cspell: ignore bevdet -->
+
+## Purpose
+
+tensorrt_bevdet is a dynamic 3D bev object detection package based on 6 surround view cameras.
+
+## Inner-workings / Algorithms
+
+BEVDet is a BEV perception algorithm based on panoramic cameras. It unifies multi-view images into the perspective of BEV for 3D object detection task. In this implementation, BEVDet network to inference with TensorRT.
+
+## Inputs / Outputs
+
+### Inputs
+
+| Name                               | Type                           | Description                         |
+| ---------------------------------- | ------------------------------ | ----------------------------------- |
+| `~/input/topic_img_fl`             | `sensor_msgs::msg::Image`      | input front_left camera image       |
```

Comment:
The input topic names use a tilde (~) which is not standard ROS2 practice.  Consider replacing `~/input/topic_img_fl` with `/input/topic_img_fl` (and similarly for all other input topics).  The use of the tilde might be interpreted as a home directory reference instead of a ROS2 topic name.